app/models/user.rb

```diff
@@ -39,6 +39,11 @@ def github_client_scopes
     GitHubUser.new(github_client, uid).client_scopes
   end
 
+  # for searching, disable pagination so we can control the number of suggestions
+  def github_search_client
```

Comment:
@johndbritton @tarebyte 

Hi @johndbritton 

What about declaring `github_client` like this:

```ruby
def github_client(opts = {})
  @github_client ||= Octokit::Client.new(opts.merge(access_token: token, auto_paginate: true))
end
```

Then we can have our `search_client` like this:

```ruby
def search_client
  @search_client ||= current_user.github_client(auto_paginate: false)
end
```